go.mod

```diff
@@ -2,6 +2,8 @@ module tailscale.com
 
 go 1.20
 
+replace github.com/coreos/go-iptables => ./go-iptables
```

Comment:
Replace directives break any repo which includes the tailscale repo as a dependency. If you really want this functionality, it needs to go upstream o go-iptables not make a copy of it here.

Reply:
I doubt that they need such feature as their purpose is to provide support for iptables instead of nothing. Most likely they propose making changes here instead to not use go-iptables when it doesn't suite. It's just that patch to avoid it, would need a lot more work, I assume.
Is there repos that actually include tailscale as dependency by the way, didn't come to think of that at all..

I am not resolving this conversation, as I didn't actually resolve the issue here. You could though feed my curiosity if you know a repo that has tailscale as dependency, I'd like to see how it's utilized.

Reply:
I'll provide another solution that does not require patching go-iptables. Hold on..

Reply:
@DentonGentry

I provided another kind of solution.

Reply:
For some reason it says that I changed file ipn/ipnlocal/local.go - never touched it... What's with that?